project/config/crc/config/config_split.config_split.production.yml

```diff
@@ -7,7 +7,8 @@ label: production
 description: ''
 weight: 0
 folder: ../project/config/crc/production
-module: {  }
+module:
+  fastly: 0
 theme: {  }
 blacklist: {  }
 graylist: {  }
```

Comment:
I think you need to add fastly.settings to the production split as well.
See project/config/publicappointmentsni/config/config_split.config_split.production.yml for reference.

Reply:
Changes done, thanks Adam